site/content/overview/about.md

```diff
@@ -0,0 +1,36 @@
+# About the AWS Deploy Tool
+
+** AWS Deploy Tool** is an interractive tooling for the .NET CLI and the AWS Toolkit for Visual Studio that helps deploy .NET applications with minimum AWS knowledge, and with the fewest clicks or commands. It works by analyzing .NET codebases and guiding developers to the right AWS service. It then selects the right deployment service, builds and packages your application, generates the right IAM roles, and creates the deployment infrastructure. It allows for a quick and easy Proof of concept (POC), smooth graduation to CI/CD, and a gradual ramp up on AWS knowledge.
+
+  > Note: AWS Deploy Tool is currently in Developer Preview.
```

Comment:
Can we remove this since we are trying to GA soon. Otherwise how are we going to remember to remove this.

Reply:
I'll remember to remove it